setup.py

```diff
@@ -126,5 +162,7 @@ def initialize_options(self):
             f"parameterized{get_version_spec('parameterized')}",
         ],
     },
-    cmdclass={"build": BuildCommand},
+    cmdclass={"build": BuildCommand, "build_ext": CMakeBuild},
```

Comment:
Structurally this looks about right. Let's hold off on this until wave has its own separate `setup.py` though. That way, the iree-turbine package remains a pure Python source package and the new wave package can be a bdist wheel.